test/good/asyncapi-very-complex-schema-output.json

```diff
@@ -0,0 +1,876 @@
+                      "ecologically": {
+                        "type": "string",
+                        "enum": [
+                          "A+++",
+                          "A++",
+                          "A+",
+                          "A",
+                          "B",
+                          "C",
+                          "D",
+                          "E",
+                          "F"
+                        ],
+                        "x-parser-schema-id": "<anonymous-schema-9>"
+                      }
+                    },
+                    "x-parser-schema-id": "<anonymous-schema-8>"
+                  }
+                ],
+                "x-parser-schema-id": "streetLightLightbulb",
+                "x-parser-schema-id-level": 2
+              },
+              "socket": {
+                "anyOf": [
+                  {
+                    "type": "object",
+                    "properties": {
+                      "country": {
+                        "type": "string",
+                        "x-parser-schema-id": "<anonymous-schema-6>"
+                      },
+                      "name": {
+                        "type": "string",
+                        "x-parser-schema-id": "<anonymous-schema-7>"
+                      }
+                    },
+                    "x-parser-schema-id": "manufactor",
+                    "x-parser-schema-id-level": 1
+                  },
+                  {
+                    "type": "object",
+                    "properties": {
+                      "demo": {
+                        "type": "string",
+                        "x-parser-schema-id": "<anonymous-schema-11>"
+                      }
+                    },
+                    "x-parser-schema-id": "<anonymous-schema-10>"
+                  }
+                ],
+                "x-parser-schema-id": "streetLightSocket",
+                "x-parser-schema-id-level": 2
+              },
+              "vendor": {
+                "oneOf": [
+                  {
+                    "type": "object",
+                    "properties": {
+                      "country": {
+                        "type": "string",
+                        "x-parser-schema-id": "<anonymous-schema-6>"
+                      },
+                      "name": {
+                        "type": "string",
+                        "x-parser-schema-id": "<anonymous-schema-7>"
+                      }
+                    },
+                    "x-parser-schema-id": "manufactor",
+                    "x-parser-schema-id-level": 1
+                  },
+                  {
+                    "type": "object",
+                    "properties": {
+                      "demo": {
+                        "type": "string",
+                        "x-parser-schema-id": "<anonymous-schema-13>"
+                      }
+                    },
+                    "x-parser-schema-id": "<anonymous-schema-12>"
```

Comment:
Code generation would generate a "class" here, which would have an anonymous schema id.

Reply:
It is a child of an oneOf. I have no clue how to solve this.